src/Counter.js

```diff
@@ -1,16 +1,16 @@
 import { React } from "react";
 
-function Counter({ clickCount, setClickCount }) {
+function Counter({ clickCount, setClickCount, label }) {
```

Comment:
I added a label prop because I want to be able to customize what the text label on this button is.